src/Authoring/WinRT.SourceGenerator/DiagnosticHelpers.cs

```diff
@@ -47,8 +47,7 @@ private void ReportDiagnostic(Diagnostic d)
             Flag();
             _context.ReportDiagnostic(d);
         }
-
-        private bool SymEq(ISymbol sym1, ISymbol sym2) { return SymbolEqualityComparer.Default.Equals(sym1, sym2); }
+        private bool SymEq(ISymbol sym1, ISymbol sym2) { return sym1.Equals(sym2); /*SymbolEqualityComparer.Default.Equals(sym1, sym2);*/ }

```

Comment:
Is this intentional?  Do you plan on going back to Equals as is?

Reply:
I went back to the original, thanks for catching this.